staging/src/slime.io/slime/modules/lazyload/install/samples/lazyload/slimeboot_cluster_prometheus.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: config.netease.com/v1alpha1
+kind: SlimeBoot
+metadata:
+  name: lazyload
+  namespace: mesh-operator
+spec:
+  image:
+    pullPolicy: Always
+    repository: docker.io/slimeio/slime-lazyload
+    tag: v0.3.0_linux_amd64
+  module:
+    - name: lazyload # custom value
+      kind: lazyload # should be "lazyload"
+      enable: true
+      general: # replace previous "fence" field
+        wormholePort: # replace to your application service ports, and extend the list in case of multi ports
+          - "80"
+          - "9080"
+      global:
+        misc:
+          globalSidecarMode: cluster # inform the mode of global-sidecar
+      metric: # indicate the metric source
+        prometheus:
+          address: http://prometheus.istio-system:9090
+          handlers:
+            destination:
+              query: |
+                sum(istio_requests_total{source_app="$source_app",reporter="destination"})by(destination_service)
+              type: Group
+  component:
+    globalSidecar:
+      enable: true
+      sidecarInject:
+        enable: true # should be true
+        # mode definition:
+        # "pod": sidecar auto-inject on pod level, need provide labels for injection
+        # "namespace": sidecar auto-inject on namespace level, no need to provide labels for injection
+        # if globalSidecarMode is cluster, global-sidecar will be deployed in slime namespace, which does not enable auto-inject on namespace level, mode can only be "pod".
+        # if globalSidecarMode is namespace, depending on the namespace definition, mode can be "pod" or "namespace".
+        mode: pod
+        labels: # optional, used for sidecarInject.mode = pod
+          sidecar.istio.io/inject: "true"
+          # istio.io/rev: canary # use control plane revisions
+      resources:
+        requests:
+          cpu: 200m
+          memory: 200Mi
+        limits:
+          cpu: 400m
+          memory: 400Mi
+      image:
+        repository: docker.io/slimeio/slime-global-sidecar
+        tag: v0.2.0-1b93bf7
```

Comment:
use `find . -type f -print0 | xargs -0 -L1 bash -c 'test "$(tail -c 1 "$0")" && echo "No new line at end of $0"'` or `pcregrep -LMr '\n\Z' .` to find all files with no newline at the end